src/utils/cost.ts

```diff
@@ -0,0 +1,11 @@
+import { CartItemType } from "../types/response";
+
+export const getOrderCost = (selectedCartItems: CartItemType[]) => {
+  return selectedCartItems.reduce((acc, cur) => {
+    return acc + cur.quantity * cur.product.price;
+  }, 0);
+};
+
+export const getDeliveryCost = (orderCost: number) => {
+  return orderCost > 100000 || orderCost === 0 ? 0 : 3000;
```

Comment:
큰 숫자를 사용할 때는 numeric separator를 이용해보시면 좋을거 같네요
```suggestion
  return orderCost > 100_000 || orderCost === 0 ? 0 : 3_000;
```

Reply:
좋아요!!

감사합니다! :)

Reply:
수정 커밋 : [chore: 큰 숫자에 numeric separator 사용](https://github.com/woowacourse/react-shopping-cart/pull/353/commits/91251e1908b96b9c4b38378abe8ec96c3d647540)